test/views/declaration/AdditionalDocumentEditViewSpec.scala

```diff
@@ -136,19 +145,63 @@ class AdditionalDocumentEditViewSpec extends UnitViewSpec with CommonMessages wi
     }
   }
 
+  "additional_document_edit view on empty page" when {
+
+    "the entered authorisation code requires additional documents" should {
```

Comment:
I can't find symetrical case for this one. Is it treated as a default? Since we have two different versions of the page I'd expect that to be reflected in two sections in tests.

Reply:
Yes. The page's default content is still there, how it was before this ticket. Only some parts (title and a couple of field hints) have to be changed when the auth code requires additional documents